tests/unit/plugins/mesos/mesos_executor_test.py

```diff
@@ -43,7 +47,10 @@ def test_creates_execution_framework_and_driver(mesos_executor):
         secret=None,
     )
 
-    mesos_executor.driver.run.assert_called_with()
+    threading.Thread.assert_called_once_with(
```

Comment:
don't use assert_called_once_with, instead check that call_args == some list

https://engineeringblog.yelp.com/2015/02/assert_called_once-threat-or-menace.html